templates/auroramysql/template.yaml

```diff
@@ -0,0 +1,1400 @@
+      be less than PreferredMaintenanceWindowEndTime.
+    Type: String
+    Default: 04:00
+    AllowedValues:
+    - 00:00
+    - 01:00
+    - 02:00
+    - 03:00
+    - 04:00
+    - 05:00
+    - 06:00
+    - 07:00
+    - 08:00
+    - 09:00
+    - '10:00'
+    - '11:00'
+    - '12:00'
+    - '13:00'
+    - '14:00'
+    - '15:00'
+    - '16:00'
+    - '17:00'
+    - '18:00'
+    - '19:00'
+    - '20:00'
+    - '21:00'
+    - '22:00'
+
+  RDSTimeZone:
+    Description: The default timezone for the database engine to use.
+    Type: String
+    Default: UTC
+    AllowedValues:
+    - Africa/Cairo
+    - Africa/Casablanca
+    - Africa/Harare
+    - Africa/Monrovia
+    - Africa/Nairobi
+    - Africa/Tripoli
+    - Africa/Windhoek
+    - America/Araguaina
+    - America/Asuncion
+    - America/Bogota
+    - America/Caracas
+    - America/Chihuahua
+    - America/Cuiaba
+    - America/Denver
+    - America/Fortaleza
+    - America/Guatemala
+    - America/Halifax
+    - America/Manaus
+    - America/Matamoros
+    - America/Monterrey
+    - America/Montevideo
+    - America/Phoenix
+    - America/Santiago
+    - America/Tijuana
+    - Asia/Amman
+    - Asia/Ashgabat
+    - Asia/Baghdad
+    - Asia/Baku
+    - Asia/Bangkok
+    - Asia/Beirut
+    - Asia/Calcutta
+    - Asia/Damascus
+    - Asia/Dhaka
+    - Asia/Irkutsk
+    - Asia/Jerusalem
+    - Asia/Kabul
+    - Asia/Karachi
+    - Asia/Kathmandu
+    - Asia/Krasnoyarsk
+    - Asia/Magadan
+    - Asia/Muscat
+    - Asia/Novosibirsk
+    - Asia/Riyadh
+    - Asia/Seoul
+    - Asia/Shanghai
+    - Asia/Singapore
+    - Asia/Taipei
+    - Asia/Tehran
+    - Asia/Tokyo
+    - Asia/Ulaanbaatar
+    - Asia/Vladivostok
+    - Asia/Yakutsk
+    - Asia/Yerevan
+    - Atlantic/Azores
+    - Australia/Adelaide
+    - Australia/Brisbane
+    - Australia/Darwin
+    - Australia/Hobart
+    - Australia/Perth
+    - Australia/Sydney
+    - Canada/Newfoundland
+    - Canada/Saskatchewan
+    - Brazil/East
+    - Europe/Amsterdam
+    - Europe/Athens
+    - Europe/Dublin
+    - Europe/Helsinki
+    - Europe/Istanbul
+    - Europe/Kaliningrad
+    - Europe/Moscow
+    - Europe/Paris
+    - Europe/Prague
+    - Europe/Sarajevo
+    - Pacific/Auckland
+    - Pacific/Fiji
+    - Pacific/Guam
+    - Pacific/Honolulu
+    - Pacific/Samoa
+    - US/Alaska
+    - US/Central
+    - US/Eastern
+    - US/East-Indiana
+    - US/Pacific
+    - UTC
+  
+  StorageEncrypted:
+    Description: Indicates whether the DB instance is encrypted.
+    Type: String
+    Default: 'true'
+    AllowedValues:
+    - 'true'
+    - 'false'
+
+  VpcId:
+    Description: The ID of the VPC to launch the RDS instance into
+    Type: AWS::EC2::VPC::Id
+
+  ServerlessMinCapacityUnit:
+    Description: The minimum capacity for an Aurora DB cluster in serverless DB engine mode. The minimum capacity must be less than or equal to the maximum capacity.
+    Type: Number
+    Default: 2
+    AllowedValues:
```

Comment:
The openshift ui doesn't like AllowedValues combined with numbers. Either change the type to `String` or remove `AllowedValues`

![2019-06-05_12-45-11](https://user-images.githubusercontent.com/20962883/58985354-fe6adc80-878f-11e9-9587-002f3a88c1a8.png)

Reply:
Changed AllowedValues from number to string.  Tested in AWS console and ok.